windows-iotcore/connect-your-device/Miracast.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Miracast on IoT Core 
+author: derekameer
+ms.author: demeer
+ms.date: 11/30/2017
+ms.topic: article
+description: Learn how to include Miracast capabilities on your device 
+keywords: windows iot, miracast, connectivity 
+---
+# Miracast on IoT Core
+
+This document will show you how to include Miracast functionality on your IoT Core device.
+
+> [!IMPORTANT]
+> This feature is only available on Insider Build 17093 and above
+
+## Miracast Overview
+
+A Miracast connection is made up of two components: the Source and the Sink. The **Miracast Source** sends content to the **Miracast Sink**, which displays the content. To create the connection, the Sink advertises itself to the connected Wi-Fi network. The Source uses the **Device Picker** to select the Sink and request a connection. Once the connection is requested, a user at the Sink receives an alert that the source is attempting to make a connection and must verify that the connection should take place. Once this happens, the Source begins casting to the Sink until either the Source cancels the connection or the Sink stops advertising.
+
+## Hardware requirements
+
+Both Source and Sink devices require Miracast-compatible Wi-Fi and Graphics drivers and chipsets to function properly. To find out if your device has Miracast-compatible hardware, run 
+```
+netsh wlan show driver
+```
+in the device's command prompt.
+
+If the device supports Miracast, you should see the output below:
+
+![Compatible Device Output](../media/Miracast/CompatibleDevice.png)
+
+The below table shows Miracast compatibility for the IoT Core reference platforms:
+
+| Board | SoC | WiFi Drivers Present | Graphics Drivers Present | Miracast-Compatible |
+|-------|-----|----------------------|--------------------------|---------------------|
+| Qualcomm Dragonboard 410c | Snapdragon 410 | Yes | Yes | Yes |
+| Raspberry Pi 2/3 | Broadcom BCM283x | Yes | No | No |
+| Minnowboard Max | Intel Atom E3825 | Yes | No | No |
+| UP Squared | Intel Celeron N3350 | Yes | Yes | Yes |
```

Comment:
Just for my own clarification, do we want to limit this to just our boards? Can this table be represented via architecture? Has Chirag weighed in on this?

Reply:
So the SoCs need to have specific drivers...and we've only tested with these ones. So I think SoC is as general as we can be, which I have here.